10-gis.Rmd

```diff
@@ -895,6 +897,70 @@ But the same is true for the lightweight SQLite/SpatiaLite database engine and G
 
 As a final note, if your data is getting too big for PostgreSQL/PostGIS and you require massive spatial data management and query performance, then the next logical step is to use large-scale geographic querying on distributed computing systems, as for example, provided by GeoMesa (http://www.geomesa.org/) or Apache Sedona [https://sedona.apache.org/; formermly known as GeoSpark - @huang_geospark_2017].
 
+
+
+
+
+## Bridges to cloud technologies and services {#cloud}
+
+
+
+### STAC, COGs, and data cubes in the cloud
+Major cloud computing providers (Amazon Web Services, Microsoft Azure / Planetary Computer, Google Cloud Platform, and others) offer huge catalogs of open Earth observation data such as the complete Sentinel-2 archive on their platforms. We can use R and directly connect to and process data from these archives, ideally from a machine in the same cloud and region.
```

Comment:
@appelmar could you add some indexes to your new sections (`\index{}` -- see examples of its use in the previous chapters)

Reply:
I've added some indexes for GDAL, STAC, COG, data cube, and cloud computing.